servicetalk-test-resources/src/test/java/io/servicetalk/test/resources/DefaultTestCertsTest.java

```diff
@@ -17,76 +17,109 @@
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.util.Base64;
 
 import static java.nio.charset.StandardCharsets.US_ASCII;
-import static java.util.stream.Collectors.joining;
-import static org.junit.jupiter.api.Assertions.assertEquals;
 
-// Not a particularly robust test, but ensures the file is found and loaded.
-// If we change algorithms/settings this size may need to change.
 class DefaultTestCertsTest {
+    private static final char[] PASSWORD = "changeit".toCharArray();
 
     @Test
     void loadServerKey() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadServerKey());
-        assertEquals(2483, contents.length());
+        loadPrivateKey(DefaultTestCerts.loadServerKey());
     }
 
     @Test
     void loadServerPem() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadServerPem());
-        assertEquals(1337, contents.length());
+        loadCertificate(DefaultTestCerts.loadServerPem());
     }
 
     @Test
     void loadServerCAPem() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadServerCAPem());
-        assertEquals(1366, contents.length());
+        loadCertificate(DefaultTestCerts.loadServerCAPem());
     }
 
     @Test
     void loadServerP12() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadServerP12());
-        assertEquals(4262, contents.length());
+        loadKeystore(DefaultTestCerts.loadServerP12());
     }
 
     @Test
     void loadClientKey() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadClientKey());
-        assertEquals(2483, contents.length());
+        loadPrivateKey(DefaultTestCerts.loadClientKey());
     }
 
     @Test
     void loadClientPem() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadClientPem());
-        assertEquals(1337, contents.length());
+        loadCertificate(DefaultTestCerts.loadClientPem());
     }
 
     @Test
     void loadClientCAPem() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadClientCAPem());
-        assertEquals(1366, contents.length());
+        loadCertificate(DefaultTestCerts.loadClientCAPem());
     }
 
     @Test
     void loadClientP12() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadClientP12());
-        assertEquals(4262, contents.length());
+        loadKeystore(DefaultTestCerts.loadClientP12());
     }
 
     @Test
     void loadTrustP12() throws Exception {
-        String contents = readFully(DefaultTestCerts.loadTruststoreP12());
-        assertEquals(2422, contents.length());
+        loadKeystore(DefaultTestCerts.loadTruststoreP12());
```

Comment:
Can you please add not-null assertions to each of these tests to verify the parsing produced a result?

```diff
diff --git a/servicetalk-test-resources/src/test/java/io/servicetalk/test/resources/DefaultTestCertsTest.java b/servicetalk-test-resources/src/test/java/io/servicetalk/test/resources/DefaultTestCertsTest.java
index 985c32070..1432edeab 100644
--- a/servicetalk-test-resources/src/test/java/io/servicetalk/test/resources/DefaultTestCertsTest.java
+++ b/servicetalk-test-resources/src/test/java/io/servicetalk/test/resources/DefaultTestCertsTest.java
@@ -33,6 +33,8 @@ import java.security.spec.PKCS8EncodedKeySpec;
 import java.util.Base64;
 
 import static java.nio.charset.StandardCharsets.US_ASCII;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
 
 class DefaultTestCertsTest {
     private static final char[] PASSWORD = "changeit".toCharArray();
@@ -79,7 +81,7 @@ class DefaultTestCertsTest {
 
     @Test
     void loadTrustP12() throws Exception {
-        loadKeystore(DefaultTestCerts.loadTruststoreP12());
+        assertThat(loadKeystore(DefaultTestCerts.loadTruststoreP12()), notNullValue());
     }
 
     private Certificate loadCertificate(final InputStream inputStream) throws CertificateException {
```